lambda/util/errorParser.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    parse: function (error, maxErrorLength) {
```

Comment:
What's the reason for a max error length?

Reply:
It's the maximum number of characters we can send to pagerduty as the incident key (we are unlikely to ever hit this limit but just in case). This is the wrong place to do the string splitting though so I've moved the string splitting to when we actually post an incident to pagerduty.